packages/dos-donts-block/src/DoDontItem.tsx

```diff
@@ -155,16 +150,15 @@ export const DoDontItem = React.forwardRef<HTMLDivElement, DoDontItemProps>(
         const memoizedRichTextEditor = useMemo(
```

Comment:
Is the memoization still needed?

Reply:
Yes unfortunately: Drag and dropping rerenders the component a LOT of times, and if we need to rerender the RTE each time it becomes a little laggy.

Reply:
Ok then its fine, i just thought with the new handling that maybe also improved